File: android/lib/src/main/java/duck/cameras/android/model/RetryException.java
package duck.cameras.android.model;

import java.util.ArrayList;

public class RetryException extends RuntimeException {
    private final ArrayList<Exception> originalExceptions;

    public RetryException(ArrayList<Exception> originalExceptions) {
        this.originalExceptions = originalExceptions;
    }

    public ArrayList<Exception> getOriginalExceptions() {
        return originalExceptions;
    }
}

File: windows/Model/RetryException.cs
using System;
using System.Collections.Generic;

namespace Duck.Cameras.Windows.Model
{
    public class RetryException : Exception
    {
        private readonly List<Exception> originalExceptions;

        public RetryException(List<Exception> originalExceptions)
        {
            this.originalExceptions = originalExceptions;
        }

        public List<Exception> getOriginalExceptions()
        {
            return originalExceptions;
        }
    }
}
